### ISSUE TITLE
Deprecation Warnings

### ISSUE THREAD

**Issue by AdamTheisen** [ARM-DOE/ACT#223]
Deprecation warnings from the latest build that we need to look into

`act/qc/clean.py:43
  /home/travis/build/ARM-DOE/ACT/act/qc/clean.py:43: DeprecationWarning: invalid escape sequence \_
    """
act/qc/clean.py:52
  /home/travis/build/ARM-DOE/ACT/act/qc/clean.py:52: DeprecationWarning: invalid escape sequence \("a value of 0 \(no bits set\) indicates the data has not "

act/tests/test_io.py:36
  /home/travis/build/ARM-DOE/ACT/act/tests/test_io.py:36: DeprecationWarning: invalid escape sequence \s act.tests.EXAMPLE_ANL_CSV, sep='\s+', column_names=headers)

act/tests/test_CropScape.py::test_cropType
  /home/travis/miniconda3/envs/testenv/lib/python3.7/site-packages/pyproj/crs/crs.py:280: FutureWarning: '+init=<authority>:<code>' syntax is deprecated. '<authority>:<code>' is the preferred initialization method. When making the change, be mindful of axis order changes: https://pyproj4.github.io/pyproj/stable/gotchas.html#axis-order-changes-in-proj-6
    projstring = _prepare_from_string(projparams)

act/tests/test_CropScape.py::test_cropType
  /home/travis/build/ARM-DOE/ACT/act/discovery/get_CropScape.py:80: DeprecationWarning: This function is deprecated. See: https://pyproj4.github.io/pyproj/stable/gotchas.html#upgrading-to-pyproj-2-from-pyproj-1
    x, y = pyproj.transform(inproj, outproj, lon, lat)`

**Comment by kenkehoe** [comment]
Yeah I don't understand these warning. The code is correct. Should I wrap these parts with a with: to capture the warnings?
